renovate.json

```diff
@@ -2,9 +2,11 @@
   "extends": ["config:base"],
   "ignorePaths": ["packages/build-field-tyes/__fixtures__/**/*"],
   "lockFileMaintenance": { "enabled": true },
-  "separateMinorPatch": true,
+  "separateMinorPatch": false,
```

Comment:
We do not need separate pull requests for minor and patch bumps, especially when they are only upgrading the lock file now.